### PR TITLE
split *.mk file into separate files per jdk version.

### PR DESCRIPTION
**Pull request by KapilPowar**
/runtimes/backlog/issues/1182

-  split *.mk file into separate files, one file per JDK_VERSION making it easy to maintain

CC: @llxia

### CODE REVIEW

**Comment by KapilPowar** [comment]
Grinder Tests
JDK 8
[Grinder 1670](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1670)
[Grinder 1669](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1669)
[Grinder 1695](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1695)

JDK 21 
[Grinder 1673](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1673 )
[Grinder 1674](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1674)

JDK 17
[Grinder 1672](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1672 )
[Grinder 1677](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1677)

JDK 11
[Grinder 1671](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1671 )
[Grinder 1692](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_JCK/1692)

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — CHANGES_REQUESTED** [review]
I want to hold off merging this one until after the main Oct release activities complete (so plan for merging in next week).

**Comment by llxia** [comment]
@KapilPowar please rebase your PR.

**Comment by KapilPowar** [comment]
Grinder Tests
JDK 8 --- Grinder 1769, Grinder 1771
JDK 11 --- Grinder 1778, Grinder 1779
JDK 21 --- Grinder 1784, Grinder 1787

File: jck/jck.mk [hunk 1]
```diff
@@ -171,239 +171,19 @@ ifeq ($(JDK_VERSION), 11)
...
-   COMPILER_LANG_STMT_TESTS_GROUP5=$(Q)lang/STMT/stmt206;lang/STMT/stmt207;lang/STMT/stmt250;lang/STMT/stmt251;lang/STMT/stmt252;lang/STMT/stmt253;lang/STMT/stmt254;lang/STMT/stmt255;lang/STMT/stmt256;lang/STMT/stmt257;lang/STMT/stmt258;lang/STMT/stmt259;lang/STMT/stmt300;lang/STMT/stmt301;lang/STMT/stmt302;lang/STMT/stmt303;lang/STMT/stmt304;lang/STMT/stmt305;lang/STMT/stmt306;lang/STMT/stmt307;lang/STMT/stmt310;lang/STMT/stmt311;lang/STMT/stmt312;lang/STMT/stmt313;lang/STMT/stmt314;lang/STMT/stmt315;lang/STMT/stmt316;lang/STMT/stmt317;lang/STMT/stmt318;lang/STMT/stmt319;lang/STMT/stmt320;lang/STMT/stmt321;lang/STMT/stmt322;lang/STMT/stmt323;lang/STMT/stmt324;lang/STMT/stmt325;lang/STMT/stmt326;lang/STMT/stmt327;lang/STMT/stmt330;lang/STMT/stmt331;lang/STMT/stmt332;lang/STMT/stmt333;lang/STMT/stmt334;lang/STMT/stmt335;lang/STMT/stmt336;lang/STMT/stmt337;lang/STMT/stmt338;lang/STMT/stmt339;lang/STMT/stmt350$(Q)
+   include $(TEST_ROOT)/jck/subdirs/jck8.mk
```
Comment (llxia) [inline jck/jck.mk:174]: Instead of having if block for all versions, can we use one line?
```
include $(TEST_ROOT)/jck/subdirs/jck$(JDK_VERSION).mk
```

**Comment by llxia** [comment]
@KapilPowar could you update the PR? Thanks

File: jck/jck.mk (hunk 1 above)
Comment (KapilPowar) [reply jck/jck.mk:174]: Yes will try suggested approach. We need to create new *.mk files for all version. 
Thanks

**Comment by KapilPowar** [comment]
Grinder Tests
JDK 8 --- Grinder 1833, Grinder 1836
JDK 11 --- Grinder 1835, Grinder 1839
JDK 17 --- Grinder 1842, Grinder 1843
JDK 21 --- Grinder 1846, Grinder 1847

**Comment by llxia** [comment]
Thanks @KapilPowar . Please remove 19 and 20.

**Comment by KapilPowar** [comment]
Grinder Tests -- 1859,1860,1861,8162

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
@ShelleyLambert Please review this PR. Thanks

**Comment by KapilPowar** [comment]
Grinder Tests
JDK 8 --- Grinder 1859, Grinder 1864
JDK 11 --- Grinder 1879
JDK 17 --- Grinder 1891
JDK 21 --- Grinder 1900

**Comment by KapilPowar** [comment]
@smlambert Please review this PR. Thanks

**Review by smlambert — APPROVED** [review]
👍

**Pull request merged**